Abductor yaml maxxing

SolsticeOfTheWinter opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
See title.

## Why / Balance
Bit jarring seeing the HYPER advanced alien shit with the same description as boring old NT stuff.

## Technical details
Literally just YAML code.

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [ X ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ X ] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl:
- tweak: Changed descriptions of Abductor items

JoeHammad1844 commented:
could you change item names from "abductor's X" to "alien X"

gluesniffler commented on Resources/Prototypes/_Shitmed/Entities/Structures/Walls/walls.yml:
```diff
@@ -3,7 +3,7 @@
   name: shuttle wall
   suffix: Diagonal
-  description: Keeps the air in and the greytide out.
+  description: Keeps the air in and the bloodthirsy salvagers out.
```
minor spelling mistake

Aidenkrz commented:
bot is right, you have spelling mistakes

SolsticeOfTheWinter commented:
> could you change item names from "abductor's X" to "alien X" please?

Changed all instances of it to "alien x" and the tools to "high tech x"
Fixed spelling mistakes aswell.

whateverusername0 reviewed (approved):
nice

gluesniffler commented:
Small nitpicks, you should probably stick to "high-tech" instead of "high-technology" since it leads to funny cropping on the UI preview of the item's name 
![image](https://github.com/user-attachments/assets/72836287-cfad-4e9f-8ffe-d2cf247f5b3e)

Besides that, you also only remade the diagonal shuttle walls description, but forgot to change the normal ones too lol
![image](https://github.com/user-attachments/assets/b04f291f-580e-4dd9-bf0e-9632a6479006)
![image](https://github.com/user-attachments/assets/075547c3-799e-4835-8af6-61d87c1ca3c6)

gluesniffler commented on Resources/Prototypes/_Shitmed/Entities/Specific/Abductor/surgery.yml:
```diff
@@ -39,7 +39,7 @@
 
 - type: entity
-  name: abductor's searing tool
+  name: high-technology searing tool
```
```suggestion
  name: high-tech searing tool

gluesniffler commented on Resources/Prototypes/_Shitmed/Entities/Specific/Abductor/surgery.yml:
```diff
@@ -75,7 +75,7 @@
 
 - type: entity
-  name: abductor's energy scalpel
+  name: high-technology energy scalpel
```
```suggestion
  name: high-tech energy scalpel

gluesniffler commented on Resources/Prototypes/_Shitmed/Entities/Specific/Abductor/surgery.yml:
```diff
@@ -111,8 +111,9 @@
 
 - type: entity
-  name: abductor's bone manipulator
+  name: high-technology bone manipulator
```
```suggestion
  name: high-tech bone manipulator

gluesniffler commented on Resources/Prototypes/_Shitmed/Entities/Specific/Abductor/tools.yml:
```diff
@@ -2,8 +2,8 @@
   id: CrowbarAbductor
-  name: abductor's crowbar
-  description: A multipurpose tool to pry open doors and fight interdimensional invaders.
+  name: high-technology crowbar
```
```suggestion
  name: high-tech crowbar

gluesniffler commented on Resources/Prototypes/_Shitmed/Entities/Specific/Abductor/tools.yml:
```diff
@@ -19,9 +19,10 @@
 
 - type: entity
-  name: abductor's wirecutter
+  name: high-technology wirecutter
```
```suggestion
  name: high-tech wirecutter

gluesniffler commented on Resources/Prototypes/_Shitmed/Entities/Specific/Abductor/tools.yml:
```diff
@@ -73,9 +74,10 @@
 
 - type: entity
-  name: abductor's screwdriver
+  name: high-technology screwdriver
```
```suggestion
  name: high-tech screwdriver

gluesniffler commented on Resources/Prototypes/_Shitmed/Entities/Specific/Abductor/tools.yml:
```diff
@@ -121,9 +123,10 @@
 
 - type: entity
-  name: abductor's wrench
+  name: high-technology wrench
```
```suggestion
  name: high-tech wrench

gluesniffler commented on Resources/Prototypes/_Shitmed/Entities/Specific/Abductor/tools.yml:
```diff
@@ -146,10 +149,10 @@
 
 - type: entity
-  name: abductor's welding tool
+  name: high-technology welding tool
```
```suggestion
  name: high-tech welding tool

gluesniffler commented on Resources/Prototypes/_Shitmed/Entities/Specific/Abductor/tools.yml:
```diff
@@ -181,10 +184,10 @@
 
 - type: entity
-  name: abductor's multitool
+  name: high-technology multitool
```
```suggestion
  name: high-tech multitool

gluesniffler commented on Resources/Prototypes/_Shitmed/Entities/Structures/Walls/walls.yml:
```diff
@@ -3,7 +3,7 @@
   name: shuttle wall
   suffix: Diagonal
-  description: Keeps the air in and the greytide out.
+  description: Keeps the air in and the bloodthirsty salvagers out.
```
Add the description to wallabductor below as well. Can't give you a suggestion since its out of the diff range.
